csharp/src/Apache.Arrow.Flight/Apache.Arrow.Flight.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>
+    <LangVersion>8.0</LangVersion>
+    <EmbedUntrackedSources>true</EmbedUntrackedSources>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Remove="Format\Flight.proto" />
```

Comment:
This shouldn't be nececessary. Just having the `<Protobuf Include="path-to\Flight.proto" />` should be enough.

Reply:
Removed